openslides/motions/migrations/0003_motion_comments.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from ..config_variables import get_config_variables
+
+
+def change_motions_comments(apps, schema_editor):
+    """
+    Index the comments fields in the config. Changing from an array to a dict with the
+    ids as keys. CHange all motions from an array for comments to a dict with the comments
+    field id as key to link motion comments and comments fields.
+    """
+    # We get the model from the versioned app registry;
+    # if we directly import it, it will be the wrong version.
+    ConfigStore = apps.get_model('core', 'ConfigStore')
+    Motion = apps.get_model('motions', 'Motion')
+
+    try:
+        config_comments_fields = ConfigStore.objects.get(key='motions_comments').value
+    except ConfigStore.DoesNotExist:
+        config_comments_fields = get_motions_comments_default()
```

Comment:
I think this is not useful. We know that the one and only value here is an empty dict.

Reply:
The problem here are custom default values in custom OS versions. I can remove this here if you wish, but if so @emanuelschuetze should copy this code to replace this commit in custon OS versions.

Reply:
Yes, please use customized migrations in customize OpenSlides builds.

Reply:
Ok